--- shinken-2.2-RC1.el7.centos.src/shinken.spec	2014-12-26 16:35:28.697000000 -0500
+++ shinken.spec	2015-01-08 14:40:48.609000000 -0500
@@ -2,12 +2,19 @@
 %global shinken_user nagios
 %global shinken_group nagios
 
+%global commit c27a6c8a9c68939fb02b652d493c72cb2b2c5344
+%global shortcommit %(c=%{commit}; echo ${c:0:7})
+%global date %(date +%%Y%%m%%d)
+%global checkout %{date}git%{shortcommit}
+
 Summary:        Python Monitoring tool
 Name:           shinken
 Version:        2.2
-Release:        RC1%{?dist}
+#Release:        RC1%{?dist}
+Release:        %{checkout}%{?dist}
 URL:            http://www.%{name}-monitoring.org
-Source0:        https://github.com/naparuba/%{name}/archive/%{name}-%{version}-RC1.tar.gz 
+#Source0:        https://github.com/naparuba/%{name}/archive/%{name}-%{version}-RC1.tar.gz
+Source0:        https://github.com/naparuba/%{name}/archive/%{commit}/%{name}-%{commit}.tar.gz
 Source1:        %{name}-arbiter.service
 Source2:        %{name}-broker.service
 Source3:        %{name}-reactionner.service
@@ -98,7 +105,8 @@ Shinken receiver daemon
 
 %prep
 
-%setup -q -n %{name}-%{version}-RC1
+#%setup -q -n %{name}-%{version}-RC1
+%setup -q -n %{name}-%{commit}
 
 # clean git files/
 find . -name '.gitignore' -exec rm -f {} \;
